azkaban-hadoop-security-plugin/src/main/java/azkaban/security/HadoopSecurityManager_H_2_0.java

```diff
@@ -334,12 +334,22 @@ private CredentialProviderWithKeyStore getCustomCredentialProvider(final Props p
 
   private void registerCustomCredential(final Props props, final Credentials hadoopCred,
       final String userToProxy, final Logger jobLogger, final String customCredentialProviderName) {
-    final CredentialProviderWithKeyStore customCredential = getCustomCredentialProvider(
+    final CredentialProvider customCredential = getCustomCredentialProvider(
             props, hadoopCred, jobLogger, customCredentialProviderName);
     final KeyStore keyStore = KeyStoreManager.getInstance().getKeyStore();
     if (keyStore != null) {
       // Containerized Execution.
```

Comment:
Nit: Add detail comment when keystore is expected. It can be used even without containerization.

Reply:
Will do